regtests/bin/matrix_ncep

```diff
@@ -164,9 +164,9 @@
 # Compile option
   if [ "$cmplOption" = 'y' ]
   then
-     opt="-c $cmplr -S -T"
+     opt="-o both -c $cmplr -S -T -N"
```

Comment:
Please revert this change.  We will not run with the -N option until all programs are available to be run with namelists, until then we'll continue the regression testing using inp files.

Reply:
Also please remove -o both.  This should be in the matrix.base not an option determined here.